src/Integration.Vsix.UnitTests/packages.lock.json

```diff
@@ -1723,7 +1724,7 @@
       "SonarLint.VisualStudio.Education": {
         "type": "Project",
         "dependencies": {
-          "Microsoft.VisualStudio.SDK": "[17.0.31902.203, )",
+          "Microsoft.VisualStudio.Sdk": "[17.0.31902.203, )",
```

Comment:
Not sure why the case keeps changing. Perhaps we are running different versions of VS, with slightly different code.